backend/src/apiserver/resource/resource_manager.go

```diff
@@ -333,7 +356,7 @@ func (r *ResourceManager) DeleteRun(runID string) error {
 	if err != nil {
 		return util.Wrap(err, "Delete run failed")
 	}
-	err = r.workflowClient.Delete(runDetail.Name, &v1.DeleteOptions{})
+	err = r.GetWorkflowClient(PodNamespace).Delete(runDetail.Name, &v1.DeleteOptions{})
```

Comment:
can it be private function? GetWorkflowClient-> getWorkflowClient

Reply:
SG

Reply:
Done